base_rest/__manifest__.py

```diff
@@ -20,6 +20,8 @@
     "assets": {
         "web.assets_backend": [
             "base_rest/static/src/scss/base_rest.scss",
+        ],
+        "web.assets_frontend": [
```

Comment:
wrong assets https://github.com/OCA/rest-framework/blob/14.0/base_rest/views/openapi_template.xml#L20

Reply:
hmm no, they are supposed to be loaded on frontend.... well, I'm too tired now. Let's check together tomorrow :stuck_out_tongue_winking_eye: